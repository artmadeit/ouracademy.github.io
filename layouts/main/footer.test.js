import Footer from './footer'
import { shallow } from 'enzyme'

test('<Footer>', () => {
  expect(shallow(<Footer />)).toMatchSnapshot()
})
