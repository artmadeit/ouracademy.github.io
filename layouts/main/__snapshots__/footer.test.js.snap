// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> 1`] = `
<footer
  className="jsx-3318118596"
>
  <div
    className="jsx-3318118596 container"
  >
    <div
      className="jsx-3318118596 row"
    >
      <div
        className="jsx-3318118596 col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1"
      >
        <ul
          className="jsx-3318118596 list-inline text-center"
        >
          <li
            key="facebook"
          >
            <a
              href="https://facebook.com/academyForUs"
              target="_blank"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-facebook fa-stack-1x fa-inverse"
                />
              </span>
            </a>
          </li>
          <li
            key="youtube"
          >
            <a
              href="https://www.youtube.com/channel/UC8W7MstX6lCJMmX1Q9WuUWA"
              target="_blank"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-youtube-play fa-stack-1x fa-inverse"
                />
              </span>
            </a>
          </li>
          <li
            key="github"
          >
            <a
              href="https://github.com/ouracademy"
              target="_blank"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-github fa-stack-1x fa-inverse"
                />
              </span>
            </a>
          </li>
          <li
            key="medium"
          >
            <a
              href="https://medium.com/ouracademy"
              target="_blank"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-medium fa-stack-1x fa-inverse"
                />
              </span>
            </a>
          </li>
        </ul>
        <p
          className="jsx-3318118596 copyright text-muted"
        >
          Copyright Â© Ouracademy 
          2018
        </p>
      </div>
    </div>
  </div>
  <JSXStyle
    css="footer.jsx-3318118596{padding:50px 0 65px;}footer.jsx-3318118596 .list-inline.jsx-3318118596{margin:0;padding:0;}footer.jsx-3318118596 .copyright.jsx-3318118596{font-size:14px;text-align:center;margin-bottom:0;}"
    styleId="3318118596"
  />
</footer>
`;
